### PR TITLE
LB-1047 cloudflare-go: support of origin's weights

### PR DESCRIPTION
**Pull request by sinduvi87**
Added weight and session affinity fields for lb.

### CODE REVIEW

**Comment by sinduvi87** [comment]
@elithrar @saljam

**Comment by elithrar** [comment]
Hey @sinduvi87 - you'll need to update the related tests as well: https://travis-ci.org/cloudflare/cloudflare-go/jobs/361301463#L522-L646

**Comment by sinduvi87** [comment]
Oops! sorry didn't see that. Will fix the test.

File: load_balancing_test.go [hunk 1]
```diff
@@ -55,7 +56,8 @@ func TestCreateLoadBalancerPool(t *testing.T) {
                 {
                   "name": "app-server-1",
                   "address": "0.0.0.0",
-                  "enabled": true
+                  "enabled": true,
+		  "weight": 1
```
Comment (elithrar) [inline load_balancing_test.go]: Indentation

**Pull request merged**